detector.go

```diff
@@ -192,6 +192,15 @@ func isWildcard(t platform.TargetMetadata) bool {
 	return t.Arch == "" && t.OS == ""
 }
 
+func hasWildcard(ts []buildpack.TargetMetadata) bool {
```

Comment:
Is it possible to add a test for this case?

Reply:
I was born to add a test for this case!